plugins/kotlin-as-java/src/main/kotlin/jvmSynthetic.kt

```diff
@@ -5,10 +5,11 @@ import org.jetbrains.dokka.model.Documentable
 import org.jetbrains.dokka.model.properties.WithExtraProperties
 
 internal fun WithExtraProperties<out Documentable>.hasJvmSynthetic(): Boolean {
+    @Suppress("TYPE_MISMATCH_WARNING_FOR_INCORRECT_CAPTURE_APPROXIMATION")
```

Comment:
There was a [a comment](https://github.com/Kotlin/dokka/pull/2972#discussion_r1174006893) about this in the draft, and I was under the impression you'd figure it out - any luck?

I personally don't feel comfortable suppressing our own API faults (see the `get()` operator implementation) without a plan, given that it will be reported as error in Kotlin 2.0